examples/regression/2-advanced-analysis/plot_both_uncertainties.py

```diff
@@ -86,20 +89,52 @@ def get_1d_data_with_normal_distrib(
         ("linear", LinearRegression()),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=degree_polyn)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
+            fit_intercept=False,
+            solver="highs",
+            )),
+    ]
+)
+
 
 # Estimating prediction intervals
-Params = TypedDict("Params", {"method": str, "cv": int})
 STRATEGIES = {
-    "jackknife_plus": Params(method="plus", cv=-1),
-    "jackknife_minmax": Params(method="minmax", cv=-1),
-    "cv_plus": Params(method="plus", cv=10),
-    "cv_minmax": Params(method="minmax", cv=10),
+    "jackknife_minmax": {"method": "minmax", "cv": -1},
+    "cv_minmax": {"method": "minmax", "cv": 10},
+    "jackknife_plus_ab": {"method": "plus", "cv": Subsample(n_resamplings=50)},
+    "conformalized_quantile_regression": {"method": "quantile", "cv": "split"},
 }
 y_pred, y_pis = {}, {}
 for strategy, params in STRATEGIES.items():
-    mapie = MapieRegressor(polyn_model, **params)
-    mapie.fit(X_train, y_train)
-    y_pred[strategy], y_pis[strategy] = mapie.predict(X_test, alpha=0.05)
+    if strategy == "conformalized_quantile_regression":
+        mapie = MapieQuantileRegressor(  # type: ignore
+            polyn_model_quant,
+            **params
+        )
+        X_train, X_calib, y_train, y_calib = train_test_split(
+            X_train,
+            y_train,
+            test_size=0.5,
```

Comment:
Prefer a smaller `test_size` of 0.2 or 0.3

Reply:
done